theme/thr/views/about/about.hbs

```diff
@@ -175,7 +175,7 @@
 			zusammen mit der <a
 				href="https://www.philso.uni-augsburg.de/lehrstuehle/schulpaed/Projekte/Digitalisierung/Lernen-4_0-in-der-Praxis/"
 				target="_blank">Universität Augsburg</a>
-			einen MOOC zu dem Thema <a href="https://lernen.cloud/" target="_blank">Lernen 4.0 -
+			einen MOOC zu dem Thema <a href="https://www.schulportal-thueringen.de/tio" target="_blank">Lernen 4.0 -
```

Comment:
We could use `TRAINING_URL` here too, right? That way we'd only need to adjust the url in the config, if it changes